### PR TITLE
Finish created task

### PR DESCRIPTION
**Pull request by trungnn142**

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -7,4 +7,19 @@ def full_title page_title
...
+  def link_to_remove_fields label, f
+    field = f.hidden_field :_destroy
+    link = link_to label, "#", onclick: "remove_fields(this)", remote: true
+    field + link
+  end
+
+  def link_to_add_fields label, f, assoc
+    new_obj = f.object.class.reflect_on_association(assoc).klass.new
+    fields = f.fields_for assoc, new_obj,child_index: "new_#{assoc}" do |builder|
+    render "#{assoc.to_s.singularize}_fields", f: builder
```
Comment (ThuBM) [inline app/helpers/application_helper.rb]: indent

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -7,4 +7,19 @@ def full_title page_title
...
+
+  def link_to_remove_fields label, f
+    field = f.hidden_field :_destroy
+    link = link_to label, "#", onclick: "remove_fields(this)", remote: true
+    field + link
+  end
+
+  def link_to_add_fields label, f, assoc
+    new_obj = f.object.class.reflect_on_association(assoc).klass.new
+    fields = f.fields_for assoc, new_obj,child_index: "new_#{assoc}" do |builder|
```
Comment (ThuBM) [inline app/helpers/application_helper.rb]: add space after new_object,

**Comment by ThuBM** [comment]
OK

File: app/controllers/supervisor/subjects_controller.rb [hunk 3]
```diff
@@ -45,7 +46,8 @@ def destroy
 
   private
   def subject_params
-    params.require(:subject).permit :name, :instruction
+    params.require(:subject).permit :name, :instruction,
+      tasks_attributes: [ :id, :name, :order, :_destroy]
```
Comment (dieunb) [inline app/controllers/supervisor/subjects_controller.rb]: space `[ :id`

**Pull request merged**